modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,138 @@
+----
+link:https://issues.redhat.com/browse/OADP-4224[OADP-4224]
+
+.Overriding imagePullPolicy is now possible with DPA
+
+Previously, {oadp-short} set the `imagePullPolicy` parameter to `Always` for all images.
+With this update, {oadp-short} checks if each image contains `sha256` or `sha512` digest, then it sets `imagePullPolicy` to `IfNotPresent`; otherwise sets it to `Always`. You can now override this policy by using the new `spec.containerImagePullPolicy` DPA field.
+link:https://issues.redhat.com/browse/OADP-4172[OADP-4172]
+
+.OADP Velero can now retry updating the restore status if initial update fails
+
+Previously, {oadp-short} Velero failed to update the restored CR status. This left the status at `InProgress` indefinitely. Components which relied on the backup and restore CR status to determine the completion would fail.
+With this update, the restore CR status for a restore will correctly proceed to `Completed` or `Failed` status.
+link:https://issues.redhat.com/browse/OADP-3227[OADP-3227]
+
+.Restoring BuildConfig Build from a different cluster is successful without any errors
+
+Previously, when performing a restore of the `BuildConfig` Build resource from a different cluster, the application generated an error on TLS verification to the internal image registry. The resulting error was `failed to verify certificate: x509: certificate signed by unknown authority` error.
+With this update, the restore of the `BuildConfig` build resources to a different cluster can proceed successfully without generating the `failed to verify certificate` error.
+link:https://issues.redhat.com/browse/OADP-4692[OADP-4692]
+
+.Restoring an empty PVC is successful
+
+Previously, downloading data failed while restoring an empty persistent volume claim (PVC). It failed with the following error:
+----
+data path restore failed: Failed to run kopia restore: Unable to load
+    snapshot : snapshot not found
+----
+With this update, the downloading of data proceeds to correct conclusion when restoring an empty PVC and the error message is not generated. 
+link:https://issues.redhat.com/browse/OADP-3106[OADP-3106]
+
+.There is no Velero memory leak in CSI and DataMover plugins
+
+Previously, a Velero memory leak was caused by using the CSI and DataMover plugins. When the backup ended, the Velero plugin instance was not deleted and the memory leak consumed memory until an `Out of Memory` (OOM) condition was generated in the Velero pod. With this update, there is no resulting Velero memory leak when using the CSI and DataMover plugins.
+link:https://issues.redhat.com/browse/OADP-4448[OADP-4448]
+
+.Post-hook operation does not start before the related PVs are released
+
+Previously, due to the asynchronous nature of the Data Mover operation, a post-hook might be attempted before the related pods' persistent volumes (PVs) were released by the Data Mover persistent volume claim (PVC). This problem would cause the backup to fail with a `PartiallyFailed` status.
```

Comment:
```suggestion
Previously, due to the asynchronous nature of the Data Mover operation, a post-hook might be attempted before the Data Mover persistent volume claim (PVC) releases the persistent volumes (PVs) of the related pods. This problem would cause the backup to fail with a `PartiallyFailed` status.
```

Reply:
A suggestion to avoid possession for inanimate objects, per https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives